publishedbranches/docs-mongocli.yaml

```diff
@@ -29,9 +30,9 @@ version:
     - '0.0.4'
     - '0.0.3'
   active:
-    - '1.21 (upcoming)'
-    - '1.20 (stable)'
-    - '1.19'
+    - '1.22 (upcoming)'
+    - '1.21 (stable)'
+    - '1.20'
```

Comment:
Hi, you are missing '1.19' here, is that intended?

Reply:
@mango-db updated. Sorry